xline-client/tests/auth.rs

```diff
@@ -50,7 +50,7 @@ async fn role_operations_should_success_in_normal_path() -> Result<()> {
     Ok(())
 }
 
-#[tokio::test]
+#[tokio::test(flavor = "multi_thread", worker_threads = 8)]
```

Comment:
Why should we specify the `worker_threads`?

Reply:
Removed